package.json

```diff
@@ -41,7 +41,7 @@
     "eslint-plugin-node": "^5.0.0",
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.1",
-    "jasmine": "^2.6.0",
+    "jasmine": "^3.1.0",
```

Comment:
`"jasmine": "^3.1.0"` really means `"jasmine": "^3.2.0"`, which you can see in the changes to `package-lock.json`; I recall jasmine@3.2.0 not working on some other packages.

If you really meant jasmine@3.1.x then please use tilde (`~`) as in:

    "jasmine": "~3.1.0",

If you are sure that jasmine@3.2.0 will work then please update as `"jasmine": "^3.2.0"` to avoid any possible confusion.

Reply:
If it passes on jasmine@3.2.0 then I would prefer option 1 (bump to `^3.2.0`).

I recall that `package-lock.json` was added to this repo in the past and is wanted in master branch of the other repos for the upcoming major release. This means that all changes to `package-lock.json` need to be committed whenever you install a new package or update.

Reply:
> There will always be confusion for those who don't understand `^`.

I would definitely agree, though understanding of semver should be a prerequisite for understanding and updating package*.json (at least at a basic level).